api/functions/carpark-event.js

```diff
@@ -0,0 +1,16 @@
+import { EventBridge } from 'aws-sdk'
```

Comment:
we're using the specific v3 package eveywhere else
```suggestion
import { EventBridge } from '@aws-sdk/client-eventbridge'
```